middlewares/content-security-policy/package-overrides.json

```diff
@@ -7,5 +7,8 @@
   ],
   "description": "Content Security Policy middleware",
   "version": "3.4.0",
-  "keywords": ["express", "security", "content-security-policy", "csp", "xss"]
+  "keywords": ["express", "security", "content-security-policy", "csp", "xss"],
```

Comment:
Could you revert this file? Helmet v5 will include a breaking change in this package, so we'll want to require Node 12.

Reply:
Sure, reverted in https://github.com/helmetjs/helmet/pull/315/commits/f1114429efa5d0cad52b0b602e1f13381904f32a